### PR TITLE
add energy plot to bubble

### PR DESCRIPTION
**Pull request by simonbyrne**
@szy21 does this look right?

### CODE REVIEW

File: examples/hybrid/bubble.jl [hunk 1]
```diff
@@ -265,7 +292,7 @@ rhs!(dYdt, Y, nothing, 0.0);
 # run!
 using OrdinaryDiffEq
 Δt = 0.03
-prob = ODEProblem(rhs!, Y, (0.0, 200.0))
+prob = ODEProblem(rhs!, Y, (0.0, 800.0))
```
Comment (jakebolewski) [inline examples/hybrid/bubble.jl:295]: this is going to timeout most likely

File: examples/hybrid/bubble.jl [hunk 2]
```diff
@@ -104,19 +104,47 @@ function init_dry_rising_bubble_2d(x, z)
...
+
+Y = Fields.FieldVector(Yc = Yc, ρw = ρw)
+
+function energy(Yc, ρu, z)
+    ρ = Yc.ρ
+    ρθ = Yc.ρθ
+    u = ρu / ρ
+    kinetic = ρ * norm(u)^2 / 2
+    potential = z * grav * ρ
+    internal = C_v * pressure(ρθ) / (ρ * R_d)
```
Comment (szy21) [inline examples/hybrid/bubble.jl:127]:
```suggestion
    internal = (C_v * pressure(ρθ) / (ρ * R_d)) * ρ
```
Sorry when you asked about internal energy I didn't realized you need to multiply it by ρ too.

Comment (szy21) [reply examples/hybrid/bubble.jl]: Yes.

File: examples/hybrid/bubble.jl [hunk 3]
```diff
@@ -186,7 +214,7 @@ function rhs!(dY, Y, _, t)
     @. dρw = hdiv(hgrad(ρw))
     Spaces.weighted_dss!(dYc)
 
-    κ = 10.0
+    κ = 3.65 # m^4/s
     @. dYc.ρ = κ * hdiv(hgrad(dYc.ρ))
```
[the remark this answers is not in the document]
Comment (akshaysridhar) [reply examples/hybrid/bubble.jl]: @szy21 Good catch, see updated file

**Comment by akshaysridhar** [comment]
@simonbyrne Collapsing changes into #279 from my working branch. We can close this.

**Pull request closed without merging**